Remove reference to other table from column classname in DC_Table

dennisbohn opened the pull request:
When I reference to another table in the list.label.fields of a dca, I get the request as column classname.

```
...
'list' => [
  'label' => [
    'fields' => [
      'send_to_customer',
      'member:tl_member.CONCAT(lastname,", ",firstname)',
      'event:tl_calendar_events.CONCAT(title, DATE_FORMAT(FROM_UNIXTIME(startTime), CHAR(32, 91, 37, 100, 46, 37, 109, 46, 37, 89, 93)))',
      'amount',
      'tstamp'
    ]
  ]
],
...
```

![image](https://user-images.githubusercontent.com/87128053/129586079-9884b1fe-7a7c-4553-ada3-878104e927e3.png)

The bugfix removes everything from the colon.

![image](https://user-images.githubusercontent.com/87128053/129588142-26caaa5f-c2f2-4412-b302-0f3623441fd6.png)

dennisbohn committed:
Remove reference to other table from column classname

When I reference to another table in the list.label.fields of a dca, I get the request as column classname.

```
...
'list' => [
  'label' => [
    'fields' => [
      'send_to_customer',
      'member:tl_member.CONCAT(lastname,", ",firstname)',
      'event:tl_calendar_events.CONCAT(title, DATE_FORMAT(FROM_UNIXTIME(startTime), CHAR(32, 91, 37, 100, 46, 37, 109, 46, 37, 89, 93)))',
      'amount',
      'tstamp'
    ]
  ]
],
...
```

![image](https://user-images.githubusercontent.com/87128053/129586079-9884b1fe-7a7c-4553-ada3-878104e927e3.png)

fritzmg commented:
Does this issue also occur in Contao 4.9? I would consider this a bugfix and thus this should be rebased to the `4.9` branch.

dennisbohn committed:
Update DC_Table.php

Got a warning, when array_shift is not used on a variable. Changed id to index 0.

dennisbohn commented:
I can reproduce the bug in Contao 4.9. Created a new pull request.

https://github.com/contao/contao/pull/3323
